packages/fxa-admin-server/package.json

```diff
@@ -28,49 +33,70 @@
   "homepage": "https://github.com/mozilla/fxa#readme",
   "readmeFilename": "README.md",
   "dependencies": {
-    "@sentry/integrations": "^5.21.1",
-    "@sentry/node": "^5.21.1",
+    "@nestjs/common": "^7.4.4",
+    "@nestjs/config": "^0.5.0",
+    "@nestjs/core": "^7.4.4",
+    "@nestjs/graphql": "^7.6.0",
+    "@nestjs/mapped-types": "^0.1.0",
+    "@nestjs/platform-express": "^7.4.4",
+    "@sentry/integrations": "^5.23.0",
+    "@sentry/node": "^5.23.0",
     "apollo-server": "^2.16.1",
     "apollo-server-express": "^2.16.1",
+    "class-transformer": "^0.3.1",
+    "class-validator": "^0.12.2",
     "convict": "^6.0.0",
     "convict-format-with-moment": "^6.0.0",
     "convict-format-with-validator": "^6.0.0",
+    "express": "^4.17.1",
+    "fxa-shared": "workspace:*",
     "graphql": "^14.6.0",
+    "graphql-tools": "^6.0.15",
```

Comment:
this version wants graphql 15. we've got 14.6 across the rest of the project, `yarn why graphql`. `^4.0.8` should work